package-lock.json

```diff
@@ -0,0 +1,15831 @@
+{
```

Comment:
Since we already have a `yarn.lock` file I'd say we should probably not also include a `package-lock.json` file. Googling around, it seems like we could run into some weird behavior if these files somehow fall out of sync with each other. Since the README recommends using yarn, I'd say we should keep what we have already and not commit this file. I'm not married to yarn though so if you feel strongly in the other direction then that's fine too. Otherwise I'd say we should throw `package-lock.json` in the `.gitignore`.
https://classic.yarnpkg.com/blog/2018/06/04/yarn-import-package-lock/

Reply:
I was using npm before switching to yarn.

I will delete it when I make my next push

Reply:
`package-lock.json` is now in the `.gitignore`, but this file was already committed so git will continue to track this file - see https://stackoverflow.com/questions/45400361/why-is-gitignore-not-ignoring-my-files